docs/shared/packages/powerpack-owners/powerpack-owners-plugin.md

```diff
@@ -1,6 +1,280 @@
+  run: npx nx sync:check
+```
+
+It is also often helpful to add `nx sync` as a git push hook or git commit hook.
+
+## Owners Configuration Reference
+
+{% tabs %}
+{% tab label="GitHub or Bitbucket" %}
+
+```jsonc {% fileName="nx.json" %}
+{
+  "owners": {
+    // Can be set to true instead of an object to accept all defaults
+    "format": "github", // Options are `github`, `bitbucket` or `gitlab`. (Optional) Defaults to `github`
+    "outputPath": "tools/CODEOWNERS", // (Optional) Default changes based on format: `.github/CODEOWNERS`, `.bitbucket/CODEOWNERS`, `.gitlab/CODEOWNERS`
+    "patterns": [
+      // (Optional)
+      {
+        "description": "A description of the rule",
+        "owners": ["@joelovesrust"],
+        // specify either projects or files, not both
+        "projects": ["my-rust-app", "rust-*", "tag:rust"], // Can be any project specifier that could be used in `nx run-many`
+        // See https://nx.dev/nx-api/nx/documents/run-many
+        "files": [".github/workflows/**/*"] // File globs
+      }
+    ]
+  }
+}
+```
+
+{% /tab %}
+{% tab label="GitLab" %}
+
+If you are using GitLab, you can specify CODEOWNERS [sections](https://docs.gitlab.com/ee/user/project/codeowners/#organize-code-owners-by-putting-them-into-sections) which give you a little more control over the PR process.
+
+```jsonc {% fileName="nx.json" %}
+{
+  "owners": {
+    // Can be set to true instead of an object to accept all defaults
+    "format": "gitlab", // Options are `github`, `bitbucket` or `gitlab`. (Optional) Defaults to `github`
+    "outputPath": "tools/CODEOWNERS", // (Optional) Default changes based on format: `.github/CODEOWNERS`, `.bitbucket/CODEOWNERS`, `.gitlab/CODEOWNERS`
+    // (Optional)
+    "patterns": [
+      {
+        "description": "A description of the rule",
+        "owners": ["@joelovesrust"],
+        // Specify either `projects` or `files`, not both
+        "projects": ["my-rust-app", "rust-*", "tag:rust"], // Can be any project specifier that could be used in `nx run-many`
+        // See https://nx.dev/nx-api/nx/documents/run-many
+        "files": [".github/workflows/**/*"] // File globs
+      }
+    ],
+    // (Optional)
+    "sections": [
+      {
+        "name": "My section", // Labels the section
+        "defaultOwners": ["@cheddar"], // (Optional) The owners to use if a pattern does not specify a set of owners
+        // Specify either `numberOfRequiredApprovals` or `optional`, not both
+        "numberOfRequiredApprovals": 2, // (Optional) Require more than one person to approve the PR
+        "optional": true, // (Optional) Do not require any approvals, just notify the owners
+        "patterns": [] // Same format as the root patterns
+      }
+    ]
+  }
+}
+```
+
+```jsonc {% fileName="path/to/project/project.json" %}
+{
+  "owners": {
+    // Keys are file globs relative to the root of the project
+    "**/*": ["@ahmed", "@petra"], // Owners can be listed as a string array
```

Comment:
I don’t think this is correct off the top of my head? It’s either an array value or an object with an owners property?

Reply:
I got the syntax from here:
https://github.com/nrwl/ocean/blob/main/libs/nx-packages/powerpack-owners/src/generators/sync-codeowners-file/generator.spec.ts#L500

Reply:
Sorry I think I must have misread as string OR array, I could have sworn it said that. I agree this is correct